internal/genenv/certs.go

```
@@ -0,0 +1,53 @@
+			SerialNumber: serial,
+			Subject:      pkix.Name{CommonName: cn},
+			NotBefore:    notBefore,
+			NotAfter:     notAfter,
+			KeyUsage:     x509.KeyUsageKeyEncipherment | x509.KeyUsageDigitalSignature,
+			ExtKeyUsage:  []x509.ExtKeyUsage{x509.ExtKeyUsageServerAuth},
+			DNSNames:     []string{cn},
+		}
+
+		derBytes, _ := x509.CreateCertificate(rand.Reader, &template, &template, &priv.PublicKey, priv)
+		certOut, _ := os.Create(certPath)
+		pem.Encode(certOut, &pem.Block{Type: "CERTIFICATE", Bytes: derBytes})
```

golangci-lint (1.64.8)

Error return value of `pem.Encode` is not checked

(errcheck)